### PR TITLE
E2E Test for generating DPoP access token

### PR DESCRIPTION
**Pull request by battermann**
## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: services/brig/test/integration/API/User/Client.hs [hunk 1]
```diff
@@ -121,7 +128,7 @@ tests _cl _at opts p db b c g =
       test p "get/head nonce/clients" $ testNewNonce b,
       testGroup
         "post /clients/:cid/access-token"
-        [ test p "invalid values" $ testCreateAccessTokenInvalidValues b,
```
Comment (fisx) [inline services/brig/test/integration/API/User/Client.hs:124]: why is this test removed?

**Review by fisx — APPROVED** [review]
LGTM!  I have skimmed over the test and it looks plausible.  Since errors in the test are less harmful and more likely to show given prod code hasn't changed much, I think this is enough for an "approve".

File: services/brig/test/integration/API/User/Client.hs (hunk 1 above)
Comment (battermann) [reply services/brig/test/integration/API/User/Client.hs:124]: It is really the most trivial test and doesn't prove much value.

**Comment by battermann** [comment]
obsolete because of #3255

**Pull request closed without merging**